.github/workflows/codeql-analysis.yml

```diff
@@ -48,7 +48,7 @@ jobs:
          cmake --build . --target OpenSSL_Target
 
       - name: Initialize CodeQL
-        uses: github/codeql-action/init@27ea8f8fe5977c00f5b37e076ab846c5bd783b96
+        uses: github/codeql-action/init@41a4ada31ba866a7f1196b9602703a89edd69e22
```

Comment:
Please undo these type of changes

Reply:
yeah, I edited windows machine which didn't have latest main.
Now fixed by rebasing to latest main